lightly/models/_momentum.py

```diff
@@ -0,0 +1,97 @@
+""" Momentum Encoder """
+
+# Copyright (c) 2020. Lightly AG and its affiliates.
+# All Rights Reserved
+
+import copy
+
+import torch
+import torch.nn as nn
+
+
+def _deactivate_requires_grad(params):
```

Comment:
this seems like a more general function than just for momentum? Should this go in a more general utilities file?

Reply:
I agree that the function can be more general but at the moment there are no other applications for it than in momentum and hence I would leave it here. We can always move it to a more general file if necessary.